console/package.json

```diff
@@ -59,7 +59,7 @@
     "deep-equal": "^1.0.1",
     "graphiql": "^0.11.11",
     "graphql": "^0.13.2",
-    "hasura-console-graphiql": "0.0.1",
+    "hasura-console-graphiql": "0.0.1-temp-analyze.0",
```

Comment:
Update to stable version after tests pass.

Reply:
Should get resolved in the latest push